css/app.css

```diff
@@ -1,3 +1,820 @@
+/* reset&global */
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  font-family: "Montserrat", sans-serif;
+}
+
 body {
-  background-color: blue;
+  background-color: #f5f5f5;
+  max-width: 1920px;
+  margin: 0 auto;
+}
+
+ul {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+section {
+  margin: 5.3125rem 0;
+}
+
+.section-title {
+  margin-bottom: 1.25rem;
+}
+
+.section-title a {
+  color: #f42;
+}
+
+/* navbar-top */
+nav {
+  display: flex;
+}
+
+.nav-logo {
+  display: none;
+  justify-content: center;
+  align-items: center;
+  background: #f42;
+  background: linear-gradient(to bottom right, #c09, #f42 85%);
+  padding: 0 1.25rem;
+}
+
+.nav-logo img {
+  height: 1rem;
+  width: 4.125rem;
+}
+
+.navbar-container {
+  flex-grow: 1;
+}
+
+nav .nav-top {
+  height: 2rem;
+  background-color: black;
```

Comment:
## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

 - [ ] Please just use `.nav-top` rule, nav is not necessary

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_